onmt/modules/multi_headed_attn.py

```diff
@@ -120,14 +120,16 @@ def unshape(x):
             return x.transpose(1, 2).contiguous() \
                     .view(batch_size, -1, head_count * dim_per_head)
 
+        query, key, value = self.linear_query(query), \
+                            self.linear_keys(key), \
+                            self.linear_values(value)
+        key = shape(key)
```

Comment:
it was 
self.linear_keys(query),\	
self.linear_values(query)

Reply:
in self attention, key,value and query are same right? So it is equivalent

Reply:
You are correct.